### ISSUE TITLE
Deprecated Names is still used in the 1.20 documentation, resulting in AttributeError.

### ISSUE THREAD

**Issue by haru0017** [pymupdf/PyMuPDF#1975]
## Describe the bug (mandatory)
Using `Document.is_stream` according to [Recipes: Low-Level Interfaces](https://github.com/pymupdf/PyMuPDF/blob/6123257f2067ea391756c0b26a809ee584651b85/docs/recipes-low-level-interfaces.rst) resulted in `AttributeError: type object 'Document' has no attribute 'is_stream'`

Later, I knew that it is a deprecated name, and using `xref_is_stream` solved the error
I haven't tried it, but the same could be said for other deprecated names

## To Reproduce (mandatory)
```
>>> print(fitz.Document.is_stream.__doc__)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'Document' has no attribute 'is_stream'

type object 'Document' has no attribute 'is_stream'
>>> print(fitz.Document.xref_is_stream.__doc__)
Check if xref is a stream object.
```

## Expected behavior (optional)
Notify that it has been deleted or update the document

## Your configuration (mandatory)
 - macOS Monterey 12.6
 - Python 3.9.13
 - PyMuPDF 1.20.3, install through pip

**Comment by JorjMcKie** [comment]
Will be included in documentation of next version.

**Comment by JorjMcKie** [comment]
>  I haven't tried it, but the same could be said for other deprecated names

You looked at section "Deprecated Names"?

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.21.0
